test/abort.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
 TESTDIR=$(dirname $0)
-${TESTDIR}/abort.lua
+${TESTDIR}/abort.lua false
 [ $? -eq 6 ]
+echo "Abort without coredump - OK"
+
+${TESTDIR}/abort.lua true
+[ $? -eq 134 ]
```

Comment:
Add a comment: 134 == 128 + 6

Reply:
Or change the check using bash arithmetics `kill -l ABRT`

Reply:
I added a comment

 btw
```
kill -l ABRT
6
```